docs/getting-started/rulesets.md

```diff
@@ -20,6 +20,7 @@ rules:
 
 Spectral has a built-in set of functions such as `truthy` or `pattern`, which you can reference in your rules.
 The example above adds a single rule that checks that tags objects have a description property defined.
+Each rule is recommended by default unless the value of recommended is explicitly to to false. 
```

Comment:
```suggestion
Since v5.0, each rule is recommended by default. Prior to that the default was to be not recommended. It is best to be explicit and set `recommended: true` or `recommended: false` if your ruleset is likely to be used across multiple versions.
```